### PR TITLE
Include cerrno

### PR DESCRIPTION
**Pull request by gdsotirov**
Include cerrno header for errno and EINTR.

### CODE REVIEW

**Comment by gdsotirov** [comment]
This is the fix for #145.

File: libproxy/url.cpp [hunk 1]
```diff
@@ -26,6 +26,7 @@
 #include <sstream> // For int/string conversion (using stringstream)
 #include <cstdio>  // For sscanf()
 #include <cstdlib>    // For atoi()
+#include <cerrno>  // For errno and EINTR
```
Comment (DimStar77) [inline libproxy/url.cpp:29]: In all other sources files where needed we have `#include <errno.h>` and I'd prefer to have the include statement aligned across the source files, not using errno.h in one place and cerrno in other areas

**Review by DimStar77 — CHANGES_REQUESTED** [review]

File: libproxy/url.cpp (hunk 1 above)
Comment (gdsotirov) [reply libproxy/url.cpp:29]: OK, but the PR was just merged. Should I create new one?

Comment (DimStar77) [reply libproxy/url.cpp:29]: > OK, but the PR was just merged. Should I create new one?

I rather meant to align things - in c++, <cerrno> is actually the preferred method imho

Comment (gdsotirov) [reply libproxy/url.cpp:29]: > I rather meant to align things - in c++, is actually the preferred method imho

Ahh, OK. Yes, in C++ it's better to use the C++ headers.

**Pull request merged**